==== triton/lib/Analysis/AxisInfo.cpp#20 - /google/src/cloud/bixia/mlir_11140cc238b8c4124e6f9efacb1601f81da096a0_1703183136/triton/lib/Analysis/AxisInfo.cpp ====
# action=edit type=text
--- triton/lib/Analysis/AxisInfo.cpp	2023-12-15 04:43:34.000000000 -0800
+++ triton/lib/Analysis/AxisInfo.cpp	2023-12-21 10:30:03.000000000 -0800
@@ -1010,7 +1010,7 @@
   knownDivisibility[0] = gcd(lb.getDivisibility(0), step.getDivisibility(0));
   auto inductionVar =
       AxisInfo(knownContiguity, knownDivisibility, knownConstancy);
-  argLattices[0]->join(inductionVar);
+  (void)argLattices[0]->join(inductionVar);
 }
 
 unsigned ModuleAxisInfoAnalysis::getPtrAlignment(Value ptr) {
